tests/test.py

```diff
@@ -10077,6 +10116,34 @@ async def test():
         loop.run_until_complete(test())
 
 
+class Psycopg3ExecutorMixin:
+    def setUp(self):
```

Comment:
hm, I see such pattern not first time. So maybe we should introduce some Base class with such approach of initializing executors?) Like `BaseExecutorInitialization` and require to override `get_executor` method with proper class or correct configuration?) We can require to set class via class variable and introduce one more method `get_executor_connection_args` if some child class needs not common args.

Reply:
I don't quite understand how you see it. Can you provide an example of usage of such executor?

Beside that, the idea is cool!